app/controllers/posts_controller.rb

```diff
@@ -11,5 +13,33 @@ def show
     @userlist = User.all
     @post = Post.find(params[:id])
     @comments = Comment.order(created_at: :desc)
+    @posts = Post.joins(:author).where(author: { id: @user.id }).order(created_at: :desc)
+  end
+
+  def new
+    @post = Post.new
+  end
+
+  def create
+    @post = Post.new
+    @post.title = params[:post][:title]
+    @post.text = params[:post][:text]
+    @post.author_id = params[:user_id]
```

Comment:
**Optional Changes**
- As a convention/coding style, Rubyist usually use a private method for params. 
```
private 
    def post_params
      params.require(:post).permit(:title, :text)
 end
```
And use it inside the CRUD methods. This helps keep your code clean and readable.